chall02/fgata-va.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+import sys
+
+def errorCode():
+    print("usage: ./xlogin.py <a-zA-Z string>")
```

Comment:
Your file won't always be named `xlogin.py`, therefore this usage is incorrect.  
You could have made it dynamic like this:

```suggestion
    print("usage: %s <a-zA-Z string>" % sys.argv[0])
```